proto/api.go

```diff
@@ -87,28 +87,27 @@ func (s stringSet) keys() []string {
 
 // AllMethods specifies the complete set of methods.
 var AllMethods = stringSet{
```

Comment:
Unrelated, but I don't see this used anywhere, and it should always be equal to the union of {Internal,Read,Write}Methods anyways.